src/pages/items/suppressors/index.js

```diff
@@ -159,11 +151,11 @@ function Suppressors(props) {
         },
         {
             title: t('Ergonomics'),
-            key: 'itemProperties.Ergonomics',
+            key: 'properties.ergonomics',
         },
         {
             title: t('Recoil'),
-            key: 'itemProperties.Recoil',
+            key: 'recoil',
```

Comment:
Does this need to be `'properties.recoil'` or is it okay?

Reply:
Both work.